components/homme/src/share/prim_driver_base.F90

```diff
@@ -1229,7 +1212,18 @@ subroutine prim_step(elem, hybrid,nets,nete, dt, tl, hvcoord, compute_diagnostic
     ! for ftype==4, also apply dynamics tendencies from forcing
     ! for ftype==4, energy diagnostics will be incorrect
     ! ===============
-    if (ftype==4) call ApplyCAMforcing_dynamics(elem,hvcoord,tl%n0,dt,nets,nete)
+    if (ftype==4) then
```

Comment:
Not sure how relevant it is but energy_halftimes() uses Q, while valid Q is only computed after remap? So, Q is not valid for ftype4 diagnostics calls and below, for HV? However, energy itself does not use Rstar, so, for energy it must be ok?